CovidTracker/Pages/Index.cshtml.cs

```diff
@@ -22,9 +22,20 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
-                string jsonString = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+                string covid_data = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+                string result = "";
```

Comment:
Added a try and catch block to avoid the error if the jsonstring is empty.
Reference:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/try-catch